docker/Dockerfile.rootless.dev

```
@@ -0,0 +1,49 @@
+        usermod -g "${target_group}" node; \
+      else \
+        groupmod -g "${USER_GID}" node; \
+      fi; \
+    fi; \
+    if [ "${USER_UID}" != "${current_uid}" ]; then \
+      usermod -u "${USER_UID}" node; \
+    fi
+
+WORKDIR /usr/src/app
+
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)